packages/vite/src/node/env.ts

```diff
@@ -0,0 +1,82 @@
+import fs from 'fs'
```

Comment:
```suggestion
import fs from 'node:fs'
```

Reply:
We haven't been using this throughout the codebase. Maybe we could do a PR with the change all at once since we've dropped node12